website/Test/FluidOrg/Convolution.purs

```diff
@@ -18,6 +18,5 @@ testFig :: T.Page -> Aff Unit
 testFig page = do
    let figId = "fig"
    waitForFigure page (figId <> "-output")
-   let toggle = figId <> "-input"
-   clickToggle page toggle
-   waitFor (T.Selector ("div#" <> toggle)) page
+   clickToggle page figId
+   waitFor (T.Selector ("div#" <> figId <> "-data-pane")) page
```

Comment:
The redundant occurrence of `figId <> “-data-pane”` suggests we should make this `waitFor` behaviour part of `clickToggle` (since that function understands what we mean by “toggle”).